protocol.md

```diff
@@ -870,6 +902,12 @@ export interface TextDocumentClientCapabilities {
 ```typescript
 interface ClientCapabilities {
 	/**
+	 * The client supports receiving the result solely through $/partialResult notifications for requests from the client to the server.
+	 * If true, the server should send all data through $/partialResult and send null as the result in the final response.
```

Comment:
@keegancsmith My point is that ordering does impact correctness when it comes to streaming $/partialResult so we need to make sure that our server does send notifications in order and that a response doesn't skip ahead of a notification that was already sent.